### PR TITLE
update video passed example 4

### PR DESCRIPTION
**Pull request by susanatx**
editorial change

Need for Final Call:

This will not require a Final Call re: editorial change

---

### **When creating PR:**

- [ ] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [ ] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [ ] Add yourself (and co-authors) as "Assignees" for PR.
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [ ] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/video-alternative-for-visual-c5a4ea.md [hunk 1]
```diff
@@ -138,7 +138,9 @@ A video element with a track element that contains descriptions.
 
 #### Passed Example 4
 
-A video element that describes some of the text on the same page. The text on the page labels the video as an alternative.
+The video element describes some of the text on the same page. The text identifies the video as an alternative.
```
Comment (Jym77) [inline _rules/video-alternative-for-visual-c5a4ea.md:141]:
```suggestion
This video element describes some of the text on the same page. The text identifies the video as an alternative.
```
We normally use demonstrative article in test cases description: https://act-rules.github.io/pages/design/rule-design/#test-cases

**Review by Jym77 — CHANGES_REQUESTED** [review]

**Review by daniel-montalvo — APPROVED** [review]

File: _rules/video-alternative-for-visual-c5a4ea.md [hunk 2]
```diff
@@ -138,7 +138,9 @@ A video element with a track element that contains descriptions.
 
 #### Passed Example 4
 
-A video element that describes some of the text on the same page. The text on the page labels the video as an alternative.
+This video element describes some of the text on the same page. The text identifies the video as an alternative.
+
```
Comment (WilcoFiers) [inline _rules/video-alternative-for-visual-c5a4ea.md:142]:
```suggestion
```

File: _rules/video-alternative-for-visual-c5a4ea.md [hunk 3]
```diff
@@ -138,7 +138,9 @@ A video element with a track element that contains descriptions.
...
+
```
Comment (WilcoFiers) [inline _rules/video-alternative-for-visual-c5a4ea.md:143]:
```suggestion
```

**Review by WilcoFiers — APPROVED** [review]

**Review by HelenBurge — APPROVED** [review]

**Comment by Jym77** [comment]
Editorial changes to example description, no need for Call for Review. Merging.

**Pull request merged**